lib/stencil-push.utils.js

```diff
@@ -188,12 +190,54 @@ utils.deleteThemesIfNecessary = async (options) => {
     return options;
 };
 
+utils.checkIfDeletionIsComplete = () => {
+    return async.retryable(
+        {
+            interval: 1000,
+            errorFilter: (err) => {
+                if (err.message === 'ThemeStillExists') {
+                    console.log(`${'warning'.yellow} -- Theme still exists;Retrying ...`);
+                    return true;
+                }
+                return false;
+            },
+            times: 5,
```

Comment:
Talked offline. After we check the deletion status 5 times, the code is going to stop checking and fail with 409. We need to make sure to surface the proper error (deletion still in process) in this case.

Reply:
thanks for the code review! @bc-jcha 

actually, it turns out the error raised is covered by a function called `printCliResultError`, so it doesn't go all the way and fail on 409.
<img width="1288" alt="image" src="https://user-images.githubusercontent.com/19962441/171727944-da8e3fd0-5b34-4769-a51b-e68dabb232dc.png">

So i think it's good to leave it as is. Or maybe we can change the error message more user friendly with better explanation.

FYI, this is how it fails when the actual 409 happens for the second time. 
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/19962441/171728209-49397220-d81d-430d-9f83-ddb1dc638796.png">